nmdc_runtime/site/translation/submission_portal_translator.py

```diff
@@ -127,6 +127,7 @@ def _get_pi(
             email=study_form.get("piEmail"),
             orcid=study_form.get("piOrcid"),
             profile_image_url=self.study_pi_image_url,
+            type=nmdc.PersonValue.class_class_curie,
```

Comment:
@sierra-moxon @turbomam  what was your recommendation today instead of class_class_curie to return the type curie?

Reply:
e.g. something like:
```
pv_details = self.schema_view.get_class("PersonValue")
pv_type = pv_details.class_uri
```

Reply:
> e.g. something like:
> 
> ```
> pv_details = self.schema_view.get_class("PersonValue")
> pv_type = pv_details.class_uri
> ```

Thanks @sierra-moxon that makes sense